airflow/dev/docker-compose-data-ingestion.yml

```diff
@@ -15,6 +21,12 @@ services:
             - webserver
         environment:
             - PORT=8080
+            # Ensure that the project set with PROJECT_ID has a BigQuery dataset named 'data_sources'
+            - DATASET_NAME=data_sources
```

Comment:
@slagathorr @mayaspivak this is related to #103 

Are we ok with this being the standard BQ schema across all projects?
<project_id>
- data_sources
-- <some_table_1>
-- <some_table_2>
--<...>
- <future_data_sources_dataset>
-- <more_tables_1>
-- <more_tables_2>
--<...>

Reply:
That works. If we don't already, we should set up a dataset called data_sources in the terraform config.

Reply:
I'd say we should probably have the datasets correspond to each major source. CDC, CDC BRFSS, Census ACS, etc., all major groups (where it makes sense) ideally would. I'd avoid just having one "data" dataset that has everything as it tends ot make discoverability more difficult down the road. The PDP does a good job at showing what that looks like.

![image](https://user-images.githubusercontent.com/10038712/99101865-36906480-25ab-11eb-9bbb-87f93227613e.png)

Reply:
No objection to separating these out into per source datasets but we'll need to refactor our ingestion server to support this.

For now i'll use the dataset name we defined in terraform so that we can get something working end to end with what we have.

Reply:
Yup in the interim that SGTM.